packages/snaps-controllers/src/snaps/SnapController.ts

```diff
@@ -985,6 +994,11 @@ export class SnapController extends BaseController<
       `${controllerName}:getFile`,
       (...args) => this.getSnapFile(...args),
     );
+
+    this.messagingSystem.registerActionHandler(
+      `${controllerName}:getManifest`,
+      (...args) => this.getSnapManifest(...args),
```

Comment:
How do you foresee this working in the clients? What kind of pattern should we use to access this data?

Reply:
I was hoping we can call this whenever we need to display some information from the manifest. But if not, we can access the localization files from state, and use the `getLocalizedSnapManifest` function directly.

Reply:
It's probably easiest if we can write it as a selector. Maybe worth starting to experiment with in the extension?